### PR TITLE
Santas Super Blocks

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 32 functions (+0.32%) on `us`.

⚠️ This PR introduces 50 warnings:
- src/evt/msg_api.c:152: warning: implicit declaration of function `set_message_images'
- src/msg.c:799: warning: passing arg 15 of `draw_box' from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:102: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:10: warning: redundant redeclaration of `dgb_04_SuperBlock_CantUpgradeMessages' in same scope
- src/world/common/atomic/SuperBlock.inc.c:10: warning: redundant redeclaration of `flo_16_SuperBlock_CantUpgradeMessages' in same scope
- src/world/common/atomic/SuperBlock.inc.c:114: warning: implicit declaration of function `dead_create_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:114: warning: implicit declaration of function `dead_create_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:11: warning: redundant redeclaration of `dgb_04_SuperBlock_PartnerIDs' in same scope
- src/world/common/atomic/SuperBlock.inc.c:11: warning: redundant redeclaration of `flo_16_SuperBlock_PartnerIDs' in same scope
- src/world/common/atomic/SuperBlock.inc.c:125: warning: implicit declaration of function `dead_hide_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:125: warning: implicit declaration of function `dead_hide_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:12: warning: redundant redeclaration of `dgb_04_SuperBlock_UpgradeDescMessages' in same scope
- src/world/common/atomic/SuperBlock.inc.c:12: warning: redundant redeclaration of `flo_16_SuperBlock_UpgradeDescMessages' in same scope
- src/world/common/atomic/SuperBlock.inc.c:133: warning: implicit declaration of function `dead_destroy_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:133: warning: implicit declaration of function `dead_destroy_popup_menu'
- src/world/common/atomic/SuperBlock.inc.c:13: warning: redundant redeclaration of `dgb_04_SuperBlock_UpgradeOrbAngles' in same scope
- src/world/common/atomic/SuperBlock.inc.c:13: warning: redundant redeclaration of `flo_16_SuperBlock_UpgradeOrbAngles' in same scope
- src/world/common/atomic/SuperBlock.inc.c:142: warning: implicit declaration of function `dead_heap_free'
- src/world/common/atomic/SuperBlock.inc.c:142: warning: implicit declaration of function `dead_heap_free'
- src/world/common/atomic/SuperBlock.inc.c:29: warning: implicit declaration of function `dead_entity_upgrade_block_hide_content'
- src/world/common/atomic/SuperBlock.inc.c:29: warning: implicit declaration of function `dead_entity_upgrade_block_hide_content'
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/common/atomic/SuperBlock.inc.c:98: warning: assignment from incompatible pointer type
- src/world/partner/goombario.c:240: warning: implicit declaration of function `cancel_current_message'

File: src/EED1E0.c [hunk 1]
```diff
@@ -6,38 +6,15 @@
...
 #define gPartnerActionStatus dead_gPartnerActionStatus2
+#define entity_upgrade_block_hide_content dead_entity_upgrade_block_hide_content
 
-#include "world/common/SetOverrideFlags_40.inc.c"
+#define create_popup_menu dead_create_popup_menu
+#define hide_popup_menu dead_hide_popup_menu
+#define destroy_popup_menu dead_destroy_popup_menu
+#define heap_free dead_heap_free
+#define wPartnerHudScripts D_80100060
+#define wDisabledPartnerHudScripts D_801000A0
```
Comment (nanaian) [inline src/EED1E0.c:16]: These symbols should probably be renamed to give them the dead_ prefix. And also go in dead.h?

**Comment by ethteck** [comment]
I'm gonna help clover with the warnings and some data issues in a little bit. I can also explain the dead_ stuff

File: src/EED1E0.c (hunk 1 above)
Comment (HailSanta) [reply src/EED1E0.c:16]: Thanks, I did some of them and held off on the rest as I wasn't exactly sure how the dead stuff is supposed to be organized.

File: src/EC9D00.c [hunk 2]
```diff
@@ -6,36 +6,6 @@
 #define NAMESPACE EC9D00
 
 #define gPartnerActionStatus dead_gPartnerActionStatus2
+#define entity_upgrade_block_hide_content dead_entity_upgrade_block_hide_content
```
Comment (ethteck) [inline src/EC9D00.c:9]: move to dead.h plz

**Review by ethteck — COMMENTED** [review]
I'll do dat

**Pull request merged**